File: app/src/main/java/vergecurrency/vergewallet/helpers/utils/TransactionUtils.java
package vergecurrency.vergewallet.helpers.utils;

import java.text.SimpleDateFormat;
import java.util.Date;

public final class TransactionUtils {
    public static final SimpleDateFormat DATE_FORMATTER = new SimpleDateFormat("dd LLLL yyyy");
    public static final SimpleDateFormat TIME_FORMATTER = new SimpleDateFormat("HH:mm");
    public static final String EXTRA_TRANSACTION = "transaction";

    public static String toFormattedDate(long milliseconds) {
        Date date = new Date(milliseconds * 1000);
        return String.join(" ", DATE_FORMATTER.format(date), "at", TIME_FORMATTER.format(date));
    }

}

File: app/src/main/kotlin/vergecurrency/vergewallet/helpers/utils/TransactionUtils.kt
package vergecurrency.vergewallet.helpers.utils

import android.annotation.SuppressLint
import android.content.Context
import android.text.format.DateFormat
import java.text.SimpleDateFormat
import java.util.*

object TransactionUtils {
    const val EXTRA_TRANSACTION = "transaction"

    fun toFormattedDate(milliseconds: Long, appContext: Context): String {
        val date = Date(milliseconds * 1000)
        return String.format("%s %s", DateFormat.getMediumDateFormat(appContext).format(date), DateFormat.getTimeFormat(appContext).format(date));
    }
}


